amr-wind/core/Field.cpp

```diff
@@ -28,6 +28,7 @@ FieldInfo::FieldInfo(
 {
     for (int i = 0; i < AMREX_SPACEDIM * 2; ++i) {
         m_bc_type[i] = BC::undefined;
+        m_bc_values_d[i] = nullptr;
```

Comment:
Would it be possible to do this in constructor initializer list?

Reply:
like this?
`    , m_bc_values(AMREX_SPACEDIM * 2, amrex::Vector<amrex::Real>(ncomp, 0.0))
    , m_bc_values_d({})
    , m_bc_values_dview(ncomp * AMREX_SPACEDIM * 2)`

Reply:
```
, m_bc_values_d(nullptr)
``` 

Since this is an AMReX data structure. I was curious if it had this option.

Reply:
it doesn't have that option :(

Reply:
Thanks for clearing that up.